app/Registry.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+namespace app;
+
+class Registry
+{
+    private static array $_storage = array();

```

Comment:
```suggestion
    private static array $storage = []
    
```
с поддеркивания не принято переменные называть (язык не запрещает, но чтобы не смешивать с названиями магических методов)

Reply:
исправил +